holidays/countries/benin.py

```diff
@@ -0,0 +1,164 @@
+        * [2024 Prophet's Birthday](https://web.archive.org/web/20250215034616/https://travail.gouv.bj/page/communiques/la-journee-du-dimanche-15-septembre-2024-jour-de-maoloud-est-feriee-chomee-et-payee-sur-toute-letendue-du-territoire-national)
+        * [2024 Christmas Day](https://web.archive.org/web/20250122081816/https://travail.gouv.bj/page/communiques/la-journee-du-mercredi-25-decembre-2024-est-feriee-chomee-et-payee-sur-toute-letendue-du-territoire-national)
+
+        * [2025](https://web.archive.org/web/20250317141957/https://beninconsulate.co.ke/index.php/news-events?start=1)
+        * [2025 New Year's Day](https://web.archive.org/web/20250422023225/https://travail.gouv.bj/page/communiques/la-journee-du-mercredi-1er-janvier-2025-fete-du-nouvel-an-est-feriee-chomee-et-payee-sur-toute-letendue-du-territoire-national)
+        * [2025 Vodoun Festival](https://web.archive.org/web/20250422034718/https://travail.gouv.bj/page/communiques/les-journees-du-jeudi-9-janvier-2024-et-vendredi-10-janvier-2024-sont-feriees-chomees-et-payees-sur-toute-letendue-du-territoire-national)
+    """
+
+    country = "BJ"
+    default_language = "fr_BJ"
+    # %s (observé).
+    observed_label = tr("%s (observé)")
+    # %s (estimé).
+    estimated_label = tr("%s (estimé)")
+    # %s (observé, estimé).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    supported_languages = ("en_US", "fr_BJ")
+    start_year = 1960
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Fête du Nouvel An"))
+
+        if self._year >= 1998:
+            # Vodoun Festival.
+            name = tr("Journée Vaudoun")
+            if self._year >= 2025:
+                self._add_observed(
+                    self._add_holiday_2nd_fri_of_jan(name),
+                    rule=FRI_TO_PREV_THU,
+                )
+            else:
+                self._add_holiday_jan_10(name)
+
+        if 1979 <= self._year <= 1989:
+            # Martyrs' Day.
+            self._add_holiday_jan_16(tr("Journée des Martyrs"))
+
+        if 1979 <= self._year <= 1989:
+            # Youth Day.
+            self._add_holiday_apr_1(tr("Journée de la Jeunesse"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Lundi de Pâques"))
```

Comment:
```suggestion
        if self._year >= 1991:
            # Easter Monday.
            self._add_easter_monday(tr("Lundi de Pâques"))
        else:
            # Easter Sunday.
            self._add_easter_sunday(tr("Jour de Pâques"))
```
If we follow the law precisely.